prep_vcfs_somatic/prep_vcf_somatic_functions.sh

```diff
@@ -572,7 +572,8 @@ function process_octopus_vcf(){
         VCF=$( decompose ${VCF} )
         VCF=$( make_vcf_upto_specs_for_VcfMerger ${VCF} )
         VCF=$( normalize_vcf ${VCF})
-        VCF=$( rename_fields_in_vcf_header_octopus_specific ${VCF} )
+        # VCF=$( rename_fields_in_vcf_header_octopus_specific ${VCF} )  # We do not need to update the header anymore since the version 0.7.4 of octopus already has the AD and
```

Comment:
Might not be a bad idea to add a check for the octopus version, I don't think we want to find out exactly which version the headers changed for, but we could say `if [[ $OCTOPUS_VERSION >= 0.7.4 ]]; then ... fi` <-- pseudo code

If this is non-trivial to find out, disregard this recommendation!